dashboard/src/main/java/eu/seaclouds/platform/dashboard/proxy/DeployerProxy.java

```diff
@@ -164,14 +120,7 @@ public TaskSummary deployApplication(String tosca) throws IOException {
      * @return String representing the sensor value
      */
     public String getEntitySensorsValue(String brooklynId, String brooklynEntityId, String sensorId) throws IOException {
-        Client client = getJerseyClient();
-        if (getUser() != null && getPassword() != null) {
-            HttpAuthenticationFeature feature = HttpAuthenticationFeature.basic(getUser(),  getPassword());
-            client.register(feature);
-        }
-
-
-        Invocation invocation = client.target(
+        Invocation invocation = getJerseyClient().target(
                 getEndpoint() + "/v1/applications/" + brooklynId + "/entities/" + brooklynEntityId + "/sensors/" + sensorId + "?raw=true")
```

Comment:
Just a suggestion, not to be addressed: I would refactor these concatenations into a variable. I prefer using String.format since the code is clearer.